packages/ui/src/components/header/NavBar.tsx

```diff
@@ -35,20 +36,45 @@ export interface NavLinkProps {
     children: ReactNode;
 }
 
-export const NavLink: FC<NavLinkProps> = ({ href, children }) => (
-    <NextLink href={href} passHref>
```

Comment:
Hi @dandheedge, 
Why are you removing the `next/link`? That means a full refresh of the page, losing the SPA effect, and forcing a reconnect of the wallet for every link in the header you click.

Reply:
@brunomenezes my bad, at first I was thinking about the style for css pseudo classes that easier applied on the `<Link>` component, didn't think through this. Going to fix it

Reply:
You still have the <Link> component; the NextLink just wraps it, so NextJS takes control over the "default" behaviour and keep the routing as a SPA.